Makefile

```
@@ -40,15 +41,13 @@ openapi-validate: render
 	prance validate --backend=openapi-spec-validator apps/api/src/hyp3_api/api-spec/openapi-spec.yml
 
 cfn-lint: render
-	cfn-lint --info --ignore-checks W3002 --template `find . -name *-cf.yml`
+	cfn-lint --info -i W3002 -i E3008 --template `find . -name *-cf.yml`
```

I prefer the long-form parameter names for readability, and it looks like there's no need to repeat the parameter flag for each entry:
```suggestion
	cfn-lint --info --ignore-checks W3002 E3008 --template `find . -name *-cf.yml`
```